### DISCUSSION TITLE
Error on build

### DISCUSSION THREAD

**Discussion by beqramo** [vercel/next.js#17700] [Help] [answered]
Hi guys,
I have a problem with building my app, it builds successfully with next js but it throws me this error, I thought first that it was my code's fault and so I revert some commits but when I saw that there was the same result so I thought that the main problem is In Vercel, it was doing everything great but 5 hours ago everything changed.

do you have any ideas, please?
```     
"next": "^9.5.3",
```

![image](https://user-images.githubusercontent.com/40486471/95387825-596f8f00-0902-11eb-8b14-76e121ed5bb6.png)

**Comment by ijjk** [comment, marked as answer]
Hi, did you change your project's settings on Vercel? It looks like it's configured for your application to be output to `build` which doesn't need to be configured unless you are doing something very custom as the default config on Vercel works for most Next.js applications.

**Reply by beqramo** [reply]
thanks that was it
